app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  after_save :update_user_counter
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_user_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
[OPTIONAL] Please rename the method name with `update_post_counter` because this is for post count not for user count so it will look more understandable :slightly_smiling_face: